doc/api/crypto.md

```diff
@@ -214,6 +219,7 @@ been completed using the [`cipher.final()`][] method.
 <!-- YAML
 added: v0.7.1
 -->
+- `auto_padding` {boolean}
```

Comment:
The `auto_padding=true` part above can be modified to `cipher.setAutoPadding([auto_padding])` if we make this item:

```
- `auto_padding` {boolean} Defaults to true.
```